tests/test_project/settings.py

```diff
@@ -0,0 +1,84 @@
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+SECRET_KEY = '_nc9_+ntllx!ptq5tzh96nn^$a#81#auhrdw#8t#+n0#4ig(lz'
```

Comment:
Per 8343ddeeb157f514cad195fd8a0e4ea25d249cdd, please use an obviously fake secret for tests.

Reply:
8fd293d ✅